Dockerfile

```diff
@@ -25,14 +25,17 @@ RUN mkdir /app && unzip /jenkinsfile-runner/vanilla-package/target/war/jenkins.w
 
 FROM openjdk:8-jdk
 ENV JENKINS_UC https://updates.jenkins.io
+ENV CASC_JENKINS_CONFIG file:///casc/jenkins.yaml
```

Comment:
Why no `/app /usr/share/jenkins/ref/casc` or so to avoid new root dirs?

Reply:
The idea was to make the path not so long to type, but seeing that the plugins are there as well, `/usr/share/jenkins/ref/casc` makes more sense - changing this.